includes/specials/SpecialManageWiki.php

```diff
@@ -159,7 +159,14 @@ private function reusableFormDescriptor( string $module, array $options ) {
 			];
 
 			$createForm = HTMLForm::factory( 'ooui', $hidden + $create, $this->getContext(), 'create' );
-			$createForm->setMethod( 'post' )->setFormIdentifier( 'create' )->setSubmitCallback( [ $this, 'reusableFormSubmission' ] )->setSubmitText( $this->msg( "managewiki-{$module}-create-submit" )->plain() )->prepareForm()->show();
+
+			$createForm->setMethod( 'post' );
+			
+			if( $module == 'namespaces' ) {
```

Comment:
This can’t be in the dedicated and clean ‘reusable form submission’ why?

Reply:
What is this URL this would result in as well?

Reply:
>This can’t be in the dedicated and clean ‘reusable form submission’ why?
 do you mean put it in the other string of everything else? I guess it can be like it was before. I can't remember why I did that but yeah, that is unneeded.

> What is this URL this would result in as well?

what do you mean?

Reply:
Why is this code not in the reusable form submission, and what is the resultant URL? It’s a fairly obvious question

Reply:
because when testing I tried putting it in reusableFormSubmission and it did not work. Because it is adding the action portion to the post request of `$createForm` to result in setting the post to https://wiki.miraheze.org/wiki/Special:ManageWiki/namespaces/id to get the post value and auto fill the namespaces on creation putting it in the reusableFormSubmission has zero effect on `$createForm` apparently and therefore did not work.

Reply:
Keeping code consistency, would appending via a URL based variable ‘?name=$text’ not be an appropriate method?

Reply:
> Keeping code consistency, would appending via a URL based variable ‘?name=$text’ not be an appropriate method?

Sure it might be and I even tried that originally, sure it works, but this way seemed so much cleaner code-wise, I believe.